src/pyEQL/utils.py

```diff
@@ -32,6 +32,31 @@ def standardize_formula(formula: str):
     return Ion.from_formula(formula).reduced_formula
 
 
+def format_solutes_dict(solute_dict: dict, units: str):
+    """
+    Formats a dictionary of solutes by converting the amount to a string with the provided units suitable for passing to
+    use with the Solution class. Note that all solutes must be given in the same units.
+
+    Args:
+        solute_dict: The dictionary to format. This must be of the form dict{str: Number}
+            e.g. {"Na+": 0.5, "Cl-": 0.9}
+        units: The units to use for the solute.
+
```

Comment:
I like how you include an example for `solute_dict`, can you also give one here?